ui/pages/settings/contact-list-tab/index.scss

```diff
@@ -53,6 +53,7 @@
   &__icon {
     margin-bottom: 16px;
     display: block;
+    color: var(--color-icon-muted);
   }
 
   &__title {
```

Comment:
Bringing contrast of contact book icon down

<img width="915" alt="Screen Shot 2022-05-19 at 4 05 41 PM" src="https://user-images.githubusercontent.com/8112138/169418460-891393e0-bd4a-4d32-9b3b-a1347e1f54c6.png">